src/core/store/modules/lines/lines.reducer.ts

```diff
@@ -79,6 +81,22 @@ const linesReducer = createReducer(linesInitialState, (builder) => {
       state.selectedPosition = position;
     })
 
+    .addCase(setUpdatedPositionData, (state, { payload: { position, lineAddress, positionObject, positions } }) => {
+      if (positionObject !== undefined) {
+        let newPositions: PositionInt[] = [];
+        positions.map((positionObj: PositionInt) => {
+          let PositionId: string = positionObj.id;
+          if (PositionId === position) {
+            return;
+          } else {
+            newPositions.push(positionObj);
+          }
+        });
+        newPositions.push({ ...positionObject });
+        //@ts-ignore
```

Comment:
what's the issue that's requiring the `@ts-ignore`?

Reply:
Type 'PositionInt[]' is not assignable to type '[]'.
  Target allows only 0 element(s) but source may have more.

Reply:
Resolved by typing all the different extended types with PositionInt instead of []